packages/back-end/src/types/Integration.ts

```diff
@@ -20,7 +20,7 @@ export class DataSourceNotSupportedError extends Error {
     this.name = "DataSourceNotSupportedError";
   }
 }
-export type MetricAggregationType = "pre" | "post" | "noWindow";
+export type MetricWindowType = "pre" | "postConversion" | "postDuration" | "noWindow";
```

Comment:
This essentially allows us to add an additional parameter (`ignoreConversionEnd`) to `addPrePostTimeFilter` without having to fork it for each child class.